src/connectors/voice-search/connectVoiceSearch.ts

```diff
@@ -21,6 +21,7 @@ export type VoiceSearchConnectorParams = {
   additionalQueryParameters?: (params: {
     query: string;
   }) => PlainSearchParameters | void;
+  createVoiceSearchHelper?: typeof builtInCreateVoiceSearchHelper;
```

Comment:
We should create an interface rather than relying on the existing code inference.

Reply:
why?

Reply:
Because it's a contract that we now need to respect internally too. It makes it obvious when we break it, and will make it easier to export.

Reply:
You wouldn't write `typeof true`, but you'll explicitly write `boolean`.

Would it make sense to allow `getState` to return a custom state shape? If so, that's another limitation of this line.